test/jdk/java/util/DoubleStreamSums/CompensatedSums.java

```diff
@@ -87,12 +87,10 @@ public void testCompensatedSums() {
 
```

Comment:
Above, if there are going to be multiple uese of Math.pow(arg, 2) in the test, I recommend defining a local private static square(double d) method for this test where square is 
double square(double arg) {return arg * arg;}
The library method pow(arg, 2) does a check for the exponent being two and does a multiple instead.

Reply:
Good call. Done.